app/routes.py

```diff
@@ -101,29 +114,17 @@ def logout():
     logout_user()
     return redirect(url_for('registration'))
 
-
-# This is not how we should be doing this and needs to be reworked to fit in the Flask conventions
 # This route points to a button which will send an email
 @app.route('/send_email_button', methods=['GET', 'POST'])
 def sending_emails():
     send = Email()
     send.send_email()
     return render_template('main.html')
```

Comment:
This does not need to be a route. This can be made a method in a file and imported then ran in the appropriate route where the button will be.